src/main/java/cc/robotdreams/StudentsGroup.java

```diff
@@ -0,0 +1,54 @@
+package cc.robotdreams;
+
+import java.util.*;
+
+public class StudentsGroup {
+
+    private Student headStudent;
+    private final List<Student> students = new ArrayList<>();
+    private final List<String> homeWork = new ArrayList<>();
+    private final Map<String, Set<Student>> completedHomeWork = new HashMap<>();
+
+    public StudentsGroup(Student headStudent) {
+        this.headStudent = headStudent;
+        students.add(headStudent);
+    }
+
+    public void changeHeadStudent(Student newHeadStudent) {
+        headStudent = newHeadStudent;
```

Comment:
What will be if `newHeadStudent` is not in `students` list?

Reply:
fixed and added samples of code usage